service-base/src/main/java/org/eclipse/hono/service/registration/RegistrationHttpEndpoint.java

```diff
@@ -59,32 +58,41 @@ protected String getEventBusAddress() {
 
     @Override
     public String getName() {
-        return RegistrationConstants.REGISTRATION_ENDPOINT;
+        return "devices";
     }
 
     @Override
     public void addRoutes(final Router router) {
 
         final String pathWithTenant = String.format("/%s/:%s", RegistrationConstants.REGISTRATION_ENDPOINT, PARAM_TENANT_ID);
-        // ADD device registration
-        router.route(HttpMethod.POST, pathWithTenant).consumes(HttpUtils.CONTENT_TYPE_JSON)
-                .handler(this::doRegisterDeviceJson);
-        router.route(HttpMethod.POST, pathWithTenant)
-                .handler(ctx -> HttpUtils.badRequest(ctx, "missing or unsupported content-type"));
+
```

Comment:
my understanding is that you want to expose the API under base URI `/devices`, right? If so, then you should use the value returned by *getName* instead of `RegistrationConstants.REGISTRATION_ENDPOINT` when creating the routes ...

Reply:
Good catch, thanks !